arclight-common/src/main/java/io/izzel/arclight/common/mixin/optimization/general/AbstractHorseEntityMixin_Optimize.java

```diff
@@ -0,0 +1,26 @@
+package io.izzel.arclight.common.mixin.optimization.general;
```

Comment:
All these optimization mixins should be in the arclight-forge/src/main/java/io/izzel/arclight/impl/mixin/optimization/stream package and all mixins should be added to mixin config.

Reply:
Updated, thank you :)